examples/java8/pom.xml

```diff
@@ -0,0 +1,14 @@
+<project>
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.github.vmware.dispatch.examples.java8.hellowithdeps</groupId>
```

Comment:
I might recommend cutting this down to just `github.vmware.dispatch.examples`

Reply:
Sure I'll make it shorter